02.calendar/calendar.rb

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+def calendar
+
+  require 'date'
+  require 'optparse'
+
+  #monthとyearの設定
```

Comment:
細かいですが、`#`の後ろはスペースを入れることが多いです。
```suggestion
  # monthとyearの設定
```

ただ、そもそもの話としてこのコメントが不要かもしれませんね。
https://qiita.com/jnchito/items/f0d90af4ed44b7484103